csm_web/scheduler/models.py

```diff
@@ -36,6 +36,7 @@ def week_bounds(date):
 
 class User(AbstractUser):
     priority_enrollment = models.DateTimeField(null=True, blank=True)
+    subscribed = models.BooleanField(default=True, help_text="Automatically set to true when created")
```

Comment:
The help text should give some more information about what the field is actually doing too.